### DISCUSSION TITLE
essay/2024/01/06/coffee-rails7

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#157] [Comments]
# essay/2024/01/06/coffee-rails7

밤을 꼴딱 새웠습니다. 커피한잔 서버를 Rails 7로 업그레이드하느라.

https://jeho.page/essay/2024/01/06/coffee-rails7.html

**Comment by ziw8** [comment]
베트남 여행을 가셨군요.
그러면.. '루왁커피한잔' 인가요? 하하
즐거운 여행 되시길..

**Reply by BenjaminKim** [reply]
하하 고맙습니다!

**Comment by hyunsangwon** [comment]
역주행 하고 있습니다. :) 내용이 간결하고 지루하지 않아 읽기 좋네요 ㅎㅎ 
즐거운 여행 되세요~! 😁

**Reply by BenjaminKim** [reply]
감사합니다! 💚

**Comment by ghkdqhrbals** [comment]
개발한 기능이 정상적으로 동작하는게 정말 행복한 것 같아요! 포기하지 않는 모습 저도 본받겠습니다! :)

**Reply by BenjaminKim** [reply]
저 잘 포기합니다..😂
이 날은 어찌 일이 잘 풀려서 다행이었어요. 기분은 날아갈 것 처럼 좋았습니다.
